.github/workflows/ci.yml

```diff
@@ -18,13 +18,16 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
+      - uses: pnpm/action-setup@v4
+        with:
+          run_install: false
```

Comment:
```suggestion
```

I don't know that you need this before you install node 🤔

Reply:
ah nice -- I guess at some point the default changed -- https://github.com/pnpm/action-setup?tab=readme-ov-file#run_install


default used to be `true` i think -- which was very annoying